### ISSUE TITLE
main.ts is missing from the TypeScript compilation

### ISSUE THREAD

**Issue by PaulRDX** [qjon/angular2-filemanager#15]
Hi guys!

So I'm trying to implement your filemanager in a project but I'm not quite sure exactly how to proceed.

I ran `npm i @rign/angular2-filemanager`

I have a CloudComponent where I want to use `<filemanager>Loading...</filemanager>`

Here is my actual code:

----------------------
cloud.component.ts:
----------------------
```
import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-cloud',
  templateUrl: './cloud.component.html',
  styleUrls: ['./cloud.component.css']
})
export class CloudComponent implements OnInit {


  constructor() {
   }

  ngOnInit() {
  }

}
```

-------------------------
cloud.component.html:
-------------------------
`<filemanager>Loading...</filemanager>`

-----------------
app.module.ts:
-----------------
```
...
import { FileManagerModule } from '@rign/angular2-filemanager/main';
import { CloudComponent } from './cloud/cloud.component';

const fileManagerConfiguration: IFileManagerConfiguration = {
  urls: {
    foldersUrl: '/api/folder',
    filesUrl: '/api/files',
    folderMoveUrl: '/api/folder/move'
  },
  isMultiSelection: true,
  mimeTypes: ['image/jpg', 'image/jpeg', 'image/png'],
  maxFileSize: 50 * 1024
};

@NgModule({
  declarations: [
    ...
  ],
  imports: [
    ...
    FileManagerModule.forRoot(fileManagerConfiguration)
  ],
  providers: [
    ...
  ],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

When I run ng serve, i've got an error: `main.ts is missing from the TypeScript compilation`

**Comment by qjon** [comment]
Hi,

Could you give me more info? This _main.ts_ is my file (from filemanager) or from your project?

**Comment by PaulRDX** [comment]
The one imported by `import { FileManagerModule } from '@rign/angular2-filemanager/main';` in app.module.ts

**Comment by qjon** [comment]
It seems like a configuration problem with new CLI. I'll check it.

**Comment by qjon** [comment]
Fixed in #18 .
